libbuteosyncfw/profile/SyncSchedule.h

```diff
@@ -105,13 +115,13 @@ class SyncSchedule
      * Day numbers are defined in Qt::DayOfWeek.
      * \return Set of week day numbers.
```

Comment:
The method description needs a small adjustment.

Reply:
Sure, fixed.